frontend/src/utils/MadLibs.ts

```diff
@@ -140,6 +141,7 @@ export const DROPDOWN_TOPIC_MAP: Record<
   asthma: 'Asthma',
   avoided_care: 'Care Avoidance Due to Cost',
   cancer_screening: 'Cancer Screening',
+  cancer_incidence: 'Cancer',
```

Comment:
can we also add an entry to the SELECTED_DROPDOWN_OVERRIDES for `cancer_incidence: "Incidence for"` ? That way the MadLib will say "Investigate rates of Incidence for Breast cancer..." rather than "Investigate rates of Cancer Breast cancer..."   We still might want to workshop this more and even write new code. Or potentially pairing the screening and disease rates the way we do for our other medication adherence measures, where the cancer type is the dropdown, and the sub topic data type is either "screening" or "incidence"